### DISCUSSION TITLE
While the SEB crash due to PC shutdown/restart, the blank screen appears in Safe Exam Browser while we launch the SEB file again (cache data are getting cleared from the location "C:/<username>/Appdata/local/SebBrowser")

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#566] [Q&A]
Hi Team,

**Scenario:**
We have attended the exam in Safe Exam Browser Offline mode. While the SEB file crash due to PC shutdown/restart during attending the exam, then the SEB shows the blank screen. So, we have tried to check the cache data. But, the SEB Browser cache data are getting cleared from the location "**C:/<username>/Appdata/local/SebBrowser**".

We are using **Offline Safe Exam Browser version 3.3.2**. So, we have tested in this version.

**We have tested the below steps:**
1. After attending some questions in Exam using SEB Browser Offline mode, we have closed the browser suddenly and
then Copied the folders from the location "C:/<username>/Appdata/local/SebBrowser" and saved the folders in a different location.
2. Now we have opened the SEB file again in Offline mode. The SEB shows the blank screen.
3. Then we went to the location " C:/<username>/Appdata/local/SebBrowser" again and Replace the current folder with the previously copied data. 
4. Then we have relaunched the SEB Browser, now this time the Exam URL is loaded, and now we can able to login in SEB offline mode. The candidate data (old data which we have copied again) is present.

We need to know that why the blank screen appears in this case when the PC/SEB crash.

Kindly check and reply back asap.

**Comment by dbuechel** [comment]
The official SEB does not provide an offline mode nor store any data under "C://Appdata/local/SebBrowser". If you're using a SEB derivative from another vendor, then you'd need to contact them directly.
